client/src/commonMain/kotlin/com/algolia/search/serialize/Routes.kt

```diff
@@ -9,8 +9,14 @@ public const val RouteABTestsV2: String = "2/abtests"
 public const val RouteRules: String = "rules"
 public const val RouteKeysV1: String = "1/keys"
 public const val RouteLogs: String = "1/logs"
-public const val RouteRecommendation: String = "1/recommendation"
-public const val RouteRecommendationV2: String = "1/strategies/personalization"
+public const val RoutePersonalization: String = "1/strategies/personalization"
+public const val RouteProfiles: String = "1/profiles"
 public const val RoutePlaces: String = "1/places"
 public const val RouteDictionaries: String = "1/dictionaries"
 public const val RouteTask: String = "1/task"
+
+@Deprecated("use RoutePersonalization instead", replaceWith = ReplaceWith("RoutePersonalization"))
+public const val RouteRecommendation: String = "1/recommendation"
```

Comment:
Does it make sense to keep this and `RouteRecommendationV2` even deprecated? If I'm not mistaken they are only used in the deleted implementations and I don't think the clients use these constants explicitly.

Reply:
Since this was initially introduced as a public API, deleting it is a breaking change